crates/bevy_input/src/gamepad.rs

```diff
@@ -399,6 +404,27 @@ impl Gamepad {
         self.info.name.as_str()
     }
 
+    /// The UUID of the game pad.
+    ///
+    /// Bevy will try to reuse IDs for game controllers across invocations of
```

Comment:
Might be worth adding a note how it manages to do this

Reply:
I removed `uuid()` entirely per suggestion.